flow-typed/npm/next_v9.x.x.js

```diff
@@ -195,6 +195,7 @@ declare module "next/document" {
   }
 
   declare export var Head: Class<React$Component<any, any>>;
+  declare export var Html: Class<React$Component<any, any>>;
```

Comment:
Would you please prefer to update the `flow-typed` definitions in the upstream? It's better than improving the definitions only for ourselves. The `$FlowFixMe` can stay until it's fixed properly.